packages/sdk/src/utils/isOpenSeaBanned.ts

```diff
@@ -9,7 +9,15 @@ import { version } from '../../package.json'
  * @returns `{tokenId: true}` A dictionary of token banned status from OpenSea.
  */
 export async function isOpenSeaBanned(ids: string[]) {
-  let url = 'https://api.opensea.io/api/v1/assets'
+  const client = getClient()
```

Comment:
I don't think we can always assume this will be used with the same chain, for example in the cart it can be overridden. I think we should keep this logic but also add the ability to override the chain by passing in a chain id.